### ISSUE TITLE
After update to 0.70.0 Rocket.Chat dont start

### ISSUE THREAD

**Issue by phillipplum** [RocketChat/Rocket.Chat#12212]
### Description:

We update rocket.chat with docker-compose pull to the latest version 0.70.0 and now rocket.chat is not available.

### Steps to reproduce:

```
docker-compose pull rocketchat
docker-compose down
docker-compose up -d
```

### Server Setup Information:

- Version of Rocket.Chat Server: 0.7.0
- Operating System: Debian
- Deployment Method: Docker
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version: 

### Relevant logs:
`docker logs 6d7fa56160f4 -f`

```
Exception in callback of async function: ReferenceError: TAPi18n is not defined
    at str.replace (packages/rocketchat:mailer/server/api.js:13:110)
    at String.replace (<anonymous>)
    at translate (packages/rocketchat:mailer/server/api.js:13:39)
    at replace (packages/rocketchat:mailer/server/api.js:27:92)
    at value (packages/rocketchat:mailer/server/api.js:69:19)
    at Settings.get (packages/rocketchat:mailer/server/api.js:44:3)
    at Object.get (packages/rocketchat:lib/lib/settings.js:32:43)
    at getTemplate (packages/rocketchat:mailer/server/api.js:42:11)
    at Module.setSettings (packages/rocketchat:mailer/server/api.js:68:2)
    at index.js (packages/rocketchat:lib/startup/index.js:3:8)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at /app/bundle/programs/server/packages/rocketchat_lib.js:18360:1
    at /app/bundle/programs/server/packages/rocketchat_lib.js:18369:3
    at /app/bundle/programs/server/boot.js:411:36
    at Array.forEach (<anonymous>)
    at /app/bundle/programs/server/boot.js:220:19
    at /app/bundle/programs/server/boot.js:471:5
    at Function.run (/app/bundle/programs/server/profile.js:510:12)
    at /app/bundle/programs/server/boot.js:470:11
Updating process.env.MAIL_URL
Starting Email Intercepter...
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Proxy_host
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Proxy_services
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_URL
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_login_style
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_label_text
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_label_color
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_color
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_server_type
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_identity_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_identity_token_sent_via
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_token_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_authorize_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_scope
```

**Comment by avs-ru** [comment]
Same. But not in docker, installed by manual:
https://rocket.chat/docs/installation/manual-installation/ubuntu/index.html#install
Updated by manual:
https://rocket.chat/docs/installation/manual-installation/ubuntu/index.html#update

Begins with the same
`Exception in callback of async function: ReferenceError: TAPi18n is not defined....`
..but ends on 
`rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_callback_url`

**Comment by geekgonecrazy** [comment]
@ggazzo maybe related to #12009 ?

**Comment by avs-ru** [comment]
> @phillipplum out of curiosity your system language is probably not set to english right?

in RC: Language - Default
in OS:
```
LANG=en_US.UTF-8
LANGUAGE=en_US:en
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

**Comment by phillipplum** [comment]
Is there any new update here?

**Comment by phillipplum** [comment]
Hello, i have to open it again :(. For me, it is not fixed in the latest 0.71.0.
My logs say:

```
Exception in callback of async function: ReferenceError: TAPi18n is not defined
    at str.replace (packages/rocketchat:mailer/server/api.js:14:110)
    at String.replace (<anonymous>)
    at translate (packages/rocketchat:mailer/server/api.js:14:39)
    at replace (packages/rocketchat:mailer/server/api.js:29:92)
    at value (packages/rocketchat:mailer/server/api.js:68:19)
    at Settings.get (packages/rocketchat:mailer/server/api.js:46:3)
    at Object.get (packages/rocketchat:lib/lib/settings.js:32:43)
    at getTemplate (packages/rocketchat:mailer/server/api.js:44:11)
    at Module.setSettings (packages/rocketchat:mailer/server/api.js:67:2)
    at index.js (packages/rocketchat:lib/startup/index.js:3:8)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at /app/bundle/programs/server/packages/rocketchat_lib.js:18387:1
    at /app/bundle/programs/server/packages/rocketchat_lib.js:18396:3
    at /app/bundle/programs/server/boot.js:411:36
    at Array.forEach (<anonymous>)
    at /app/bundle/programs/server/boot.js:220:19
    at /app/bundle/programs/server/boot.js:471:5
    at Function.run (/app/bundle/programs/server/profile.js:510:12)
    at /app/bundle/programs/server/boot.js:470:11
Updating process.env.MAIL_URL
Starting Email Intercepter...
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Facebook_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Google_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Github_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Linkedin_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Meteor_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Twitter_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Proxy_host
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Proxy_services
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_URL
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_login_style
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_label_text
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_label_color
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Dolphin_button_color
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Drupal_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_GitHub_Enterprise_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Gitlab_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Tokenpass_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_id
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_secret
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_callback_url
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_server_type
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_identity_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_identity_token_sent_via
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_token_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_authorize_path
rocketchat:lib ➔ oauth_updated Accounts_OAuth_Wordpress_scope
```
